# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/test/jsmodules/chrome.toml
# Commit: 77947c04dcbe
# Full Hash: 77947c04dcbeecd5bec830882690d0180f107d7f
# Author: Yoshi Cheng-Hao Huang <allstars.chh@gmail.com>
# Date: 2025-09-24 18:03 +0000
# Regressor Bug: 1968890
# File Overlap Count: 3
# Overlapping Files: dom/base/test/jsmodules/chrome.toml, dom/base/test/jsmodules/import_404.mjs, dom/base/test/jsmodules/test_dynamicImportErrorMessage.html
# Description:
#   Bug 1968890 - Part 9: Add test for dynamic import. r=jonco
# ==============================================================================

diff -r cc48926e1974 -r 77947c04dcbe dom/base/test/jsmodules/chrome.toml
--- a/dom/base/test/jsmodules/chrome.toml	Wed Sep 24 18:03:30 2025 +0000
+++ b/dom/base/test/jsmodules/chrome.toml	Wed Sep 24 18:03:31 2025 +0000
@@ -1,6 +1,7 @@
 [DEFAULT]
 support-files = [
   "ambiguous_export.mjs",
+  "import_404.mjs",
   "import_ambiguous.mjs",
   "import_ambiguous_indirect_export.mjs",
   "import_ambiguous_export.mjs",